### ISSUE TITLE
A question about your paper

### ISSUE THREAD

**Issue by pyni** [jonbarron/robust_loss_pytorch#33]
Hi, Thanks for your amazing loss function.
After reading your papers, I have the following question:
![image](https://user-images.githubusercontent.com/18031767/166656580-f7d3383a-f16c-425f-b369-bd6dc86afec2.png)
![image](https://user-images.githubusercontent.com/18031767/166657032-af12649b-12ff-4237-a51f-91cdc17d06ce.png)

So how to get this equation? I find it looks like softmax function and why c is set to 1 here. How to ensure that this equation can make a good tradeoff between α and training weights.(Actually I can hardly find the proof in your paper. Could you please give some suggestions?)
Thanks.
Moreover, in your codes, I can not find μ in https://github.com/jonbarron/robust_loss_pytorch/blob/master/robust_loss_pytorch/general.py.
It should be ρ (x − μ, α, c), but this function is ρ (x, α, c)
